spaces/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+from users.utils      import login_decorator
+from django.http      import JsonResponse
+from django.views     import View
 
-# Create your views here.
+from spaces.models import *
+
+class HostPostView(View):
```

Comment:
```suggestion
class HostView(View):
```
host 자원과 관련된 다른 method들도 추가 될 수 있습니다. View이름을 지을 때는 어떤 자원을 활용할 건지 고려해서 지으시고, 행동과 관련해서는 method로 표현해 주시면 될 것 같습니다.